Frontend/ui-customer/src/Flow.purs

```diff
@@ -1413,12 +1430,196 @@ helpAndSupportScreenFlow = do
       modifyScreenState $ MyRideScreenStateType (\myRidesScreen -> myRidesScreen { data{offsetValue = 0}})
       myRidesScreenFlow false
     UPDATE_STATE updatedState -> do
-      modifyScreenState $ HelpAndSupportScreenStateType (\helpAndSupportScreen -> updatedState)
+      modifyScreenState $ HelpAndSupportScreenStateType (\_ -> updatedState)
       helpAndSupportScreenFlow
     DELETE_USER_ACCOUNT updatedState -> do
       _ <- Remote.sendIssueBT (Remote.makeSendIssueReq (Just updatedState.data.email) Nothing "Request To Delete Account" updatedState.data.description )
       modifyScreenState $ HelpAndSupportScreenStateType (\helpAndSupportScreen -> helpAndSupportScreen { props{showDeleteAccountView = true}, data {accountStatus = DEL_REQUESTED}})
       helpAndSupportScreenFlow
+    REPORTED_ISSUE_SCREEN updatedState -> do
+       modifyScreenState $ HelpAndSupportScreenStateType (\_ -> updatedState)
+       helpAndSupportScreenFlow
+    RESOLVED_ISSUE_SCREEN updatedState -> do
+      modifyScreenState $ HelpAndSupportScreenStateType (\_ -> updatedState)
+      helpAndSupportScreenFlow
+    RIDE_SELECTION_SCREEN selectedCategory -> do
+      modifyScreenState $ RideSelectionScreenStateType (\rideHistoryScreen -> rideHistoryScreen { data {offsetValue = 0}, selectedCategory = selectedCategory } )
+      rideSelectionScreenFlow
+    ISSUE_CHAT_SCREEN selectedCategory -> do      
+      let language = fetchLanguage $ getValueToLocalStore LANGUAGE_KEY
+      (GetOptionsRes getOptionsRes) <- Remote.getOptionsBT language selectedCategory.categoryId "" ""
+      let getOptionsRes' = (mapWithIndex (\index (Option x) ->
```

Comment:
`      let options' = (mapWithIndex (\index (Option x) ->
        { option : (show (index + 1)) <> ". " <>
                   if (language == "en")
                    then capitalizeFirstChar x.option
                    else x.option
        , issueOptionId : x.issueOptionId
        , label : x.label
        }
      ) issueInfoRes.options)`
  
  This code is used at multiple places, Better to create function and then use